src/main/java/dk/cs/aau/huppaal/abstractions/Edge.java

```diff
@@ -557,7 +573,32 @@ public void deserialize(final JsonObject json, final Component component) {
 
     @Override
     public String generateNearString() {
-        String result = "Edge";
+        return "[%s](edge:%s/%s)".formatted(generateFromToString(), parent.getName(), uuid.toString());
+    }
+
+    private String generateFromToString() {
+        return "%s -> %s".formatted(getSourceName(), getTargetName());
+    }
+
+    private String getSourceName() {
+        if(getSourceLocation() != null)
+            return getSourceLocation().getId();
+        if(getSourceJork() != null)
+            return getSourceJork().getId();
+        return getSourceCircular().toString();
+    }
+
+    private String getTargetName() {
+        if(getTargetLocation() != null)
+            return getTargetLocation().getId();
+        if (getTargetJork() != null)
+            return getTargetJork().getId();
+        return getTargetCircular().toString();
+    }
+
+    @Deprecated
+    private String generateNearStringOld() {
+        var result = "Edge";
```

Comment:
I assume that this is for the deprecated tabpanes, but I think even those could use the new one?

Reply:
Removed in c88e493d2279e39df4b58fe527896ac193d4fbfa